### DISCUSSION TITLE
Feature

### DISCUSSION THREAD

**Discussion by Valocai** [Paymenter/Paymenter#361] [Ideas]
See last page user on

**Comment by Valocai** [comment]
Adding more. Ability to see as an admin what page a user was last on. Example payment or specific product

**Reply by finnjosten** [reply]
Like can you explain why this would be a handy feature? like i dont see any purpose in it besides having to store a bunch of data

**Reply by Jimster4312** [reply]
You could do it like a "Most Visited Page" or "Most Popular Page" / Product.
Like so admins can see the most popular product viewed and most popular product bought.

Otherwise I agree it would just have to store a bunch of data.

**Reply by finnjosten** [reply]
I get what you mean but then why see what the "last" page was, wouldn't it be better to just see a list of pages with their view count and you can sort it on a bunch of filters and stuff. So you can sort by "most popular" or things like that.

I dont see the point why you would want to view the last page an user was on

**Reply by Jimster4312** [reply]
Yea thats what i meant lol, my explaining skills are...
unexplainable :)

**Reply by Valocai** [reply]
Yea pretty much.

**Reply by finnjosten** [reply]
> Yea thats what i meant lol, my explaining skills are...
> unexplainable :)

I understood what you meant just not the original authors suggestions. As just saving the last view would be a waste of storage I do support the most viewed idee and stuff
